src/config-as-code/repo-config-service.ts

```diff
@@ -0,0 +1,55 @@
+import YAML from "yaml";
+import GitHubClient from "../github/client/github-client";
+import { getCloudInstallationId } from "../github/client/installation-id";
+import { getLogger } from "../config/logger";
+import { RepoConfig } from "./repo-config";
+import RepoConfigDatabaseModel from "./repo-config-database-model";
+
+const logger = getLogger("services.config-as-code");
+const CONFIG_PATH = "aconfig.yml";
+
+const getRepoConfig = async (githubInstallationId: number, owner: string, repo: string): Promise<string | null> => {
+	const client = new GitHubClient(getCloudInstallationId(githubInstallationId), logger);
+	try {
+		const response = await client.getRepositoryContent(owner, repo, CONFIG_PATH)
+		const contentString = Buffer.from(response.data.content, "base64").toString("ascii"); 
+		logger.debug(`Converted ${response.data.content} to ${contentString}`);
+		return contentString;
+	} catch (err) {
+		logger.error(err, "Get repo failed.");
+		return null;
+	}
+}
+
+const convertYamlToRepoConfig = (input: string): RepoConfig => {
+	return YAML.parse(input)
```

Comment:
Can we make sure here somehow that the returned JSON only contains the fields that we expect? We don't want to store garbage in the database when someone decides to put (valid) garbage YAML in their config file.